### DISCUSSION TITLE
Performance when enable html attribute mode

### DISCUSSION THREAD

**Discussion by nhattran998** [unocss/unocss#276] [Q&A]
Thanks for the awesome library antfu <3, but I am curious about the performance of this tool when comparing the CSS tool like Tailwind or WindiCss.

Example: 
`<h1 class="text-red-200 font-bold .............">Hello</h1>`

compare performance with `<h1 text-red-200 font-bold ...>Hello</h1>`

**Comment by antfu** [comment]
The build time performance bench is already provided so I guess you are asking about runtime performance. The runtime performance technically is the browser's job and it's out-of-scope of UnoCSS, but given that browser's parsing and rendering is written in native code, I don't expect that would affect the performance much. I have no idea how to benchmark it properly, if anyone really wants to know that, feel free to give a try to bench and maybe share with us. Thanks.
